### DISCUSSION TITLE
What is Optimium and Validium in rollups?

### DISCUSSION THREAD

**Discussion by vasanthmarimuth** [Cyfrin/foundry-full-course-cu#2330] [Q&A]
![Screenshot 2024-08-27 161814](https://github.com/user-attachments/assets/b64e3f56-b6fe-4044-af96-c1ce2a3c01e4)
Kindly explain this or if there is any article about this leave a link below.

**Comment by EngrPips** [comment]
Hello @vasanthmarimuth, I think this article might be a good start https://www.tekedia.com/understanding-zkrollups-optimistic-rollups-and-validium/#:~:text=Validium%20is%20a%20hybrid%20between,the%20L1%20as%20validity%20proofs.

**Reply by vasanthmarimuth** [reply]
Hello @EngrPips , is optimium and optimistic rollups are same?

**Reply by EngrPips** [reply]
Hello @vasanthmarimuth, I don't know, as i haven't done any research on this topic except for finding this article link for you. Please read the article and see if it touches on what you want.

**Reply by vasanthmarimuth** [reply]
Thankyou for your valuable time @EngrPips .

**Reply by EngrPips** [reply]
You are welcome sir.
